api/envoy/config/trace/v3/zipkin.proto

```diff
@@ -68,4 +68,11 @@ message ZipkinConfig {
   // Optional hostname to use when sending spans to the collector_cluster. Useful for collectors
   // that require a specific hostname. Defaults to :ref:`collector_cluster <envoy_v3_api_field_config.trace.v3.ZipkinConfig.collector_cluster>` above.
   string collector_hostname = 6;
+
+  // If this is set to true, then Envoy will be treated as independent hop in trace chain. Server span will be created
+  // for downstream request and client span will be created for upstream request.
+  // If this is set to true, then the
+  // :ref:`start_child_span of router <envoy_v3_api_field_extensions.filters.http.router.v3.Router.start_child_span>`
+  // SHOULD be set to true also to ensure the correctness of trace chain.
```

Comment:
I still don't fully grok this, I guess this is because I don't have a good understanding of tracing and zipkin.
If this is highly recommended to set the `start_child_span` to true, then let's add a note here, so a user can easily configure their proxy.

Reply:
Got it. It's a little like the skywaling tracer. Becasuse spans of zipkin have type (client or server), and span should appear in pairs. So, a server span shoud be `root span` or `child span of one client span`. And `start_child_span` is necessary in that case.